src/translation/translation_fr_FR.ts

```diff
@@ -11,12 +11,12 @@
     <message>
         <location filename="../util.cpp" line="597"/>
         <source>(build)</source>
-        <translation type="unfinished"></translation>
+        <translation>(construction)</translation>
```

Comment:
Yay!

Reply:
(build) as it is about compiling the sources of a program, we should translate in french using "(compilation)". To build = compiler.